template/src/pageComponents/buy-me-coffee/Introduction.tsx

```diff
@@ -0,0 +1,27 @@
+import Image from 'next/image';
+import UserAvatar from './Avatar.svg';
+
+export default function Introduction() {
+  const coffeeReceivedCount = 90;
+  const userName = '0xPudgy';
+  const aboutUser = 'I am a content creator producing videos about NFTs!';
+
+  return (
+    <section>
+      <div className="mx-[-100vw] my-10 h-px bg-zinc-400 bg-opacity-20 md:my-16" />
```

Comment:
`mx-[-100vw]` can we be more elegant than this? @ppiyush13 @arsood thoughts?

Reply:
@Zizzamia  As per Figma designs, we are required to a horizontal line that spans the entire screen. However, all components on this page are nested within a `main` element, which introduces a padding of `32px`. To compensate this, I have applied a negative margin to the line, ensuring it extends across the entire screen on all viewports.

An alternative solution could involve creating a pseudo-element that spans the entire screen. I opted for the previous solution as it avoids unnecessary bloatware.

```css
/* solution 2 creating pseudo-element */
hr {
  padding: 50px 0;
  border: none;

  &:before {
    // full-width divider
    content: "";
    display: block;
    position: absolute;
    right: 0;
    max-width: 100%;
    width: 100%;
    border: 1px solid grey;
  }
}
```